glide.lock

```diff
@@ -1,5 +1,5 @@
-hash: 079b9f1fdc9ed811bab54c891e0d0b03b68e23f6428be046b0f7d87148a908be
-updated: 2019-03-25T15:56:19.152737147-04:00
+hash: 38c293cd395b48a47f66c9434e2e7b1b19aa388c37259a06d772e6ceaca4c1cc
```

Comment:
I'm not sure about the `hash` value as it might be calculated from the content of the `glide.yaml` file which might be different for `3.11` branch. But it seems like this value is not used that much https://github.com/Masterminds/glide/issues/197

Reply:
it's just a checksum afaik